src/server/api/init/getAssessment.ts

```diff
@@ -0,0 +1,21 @@
+import { Response } from 'express'
+
+import { InitRequest } from '@meta/api/request'
+
+import { AssessmentController } from '@server/controller/assessment'
+import { SettingsController } from '@server/controller/settings'
+import Requests from '@server/utils/requests'
+
+export const getAssessment = async (req: InitRequest, res: Response) => {
```

Comment:
I would maybe see if we can do something like getAssessmentWithCycle that would return 
```
{ cycle, assessment }
```

If param empty for cycle, return assessment default cycle?